sdk/clientcore/core/src/main/java/io/clientcore/core/util/serializer/ObjectSerializer.java

```diff
@@ -11,7 +11,13 @@
 /**
  * Generic interface covering serializing and deserialization objects.
  */
-public interface ObjectSerializer {
+public abstract class ObjectSerializer {
```

Comment:
Why the change to abstract class here

Reply:
This can be changed back, was playing around with a few things and at one point abstract was what I needed.